PW35_2021_Virtual/Projects/PyTorchIntegration/README.md

```diff
@@ -0,0 +1,53 @@
+Back to [Projects List](../../README.md#ProjectsList)
+
+# Integration of PyTorch and Slicer
+
+## Key Investigators
+
+- Fernando Pérez-García (University College London & King's College London)
+- Andrés Díaz-Pinto (King's College London)
+- Andras Lasso (PerkLab, Queen's University)
+
+# Project Description
+
+<!-- Add a short paragraph describing the project. -->
+
+Investigate the potential issues faced by users who would like to use a trained
+convolutional neural network (deep learning model) inside Slicer, using PyTorch.
+
+## Objective
+
+<!-- Describe here WHAT you would like to achieve (what you will have as end result). -->
+
+Issues that will be addressed:
+
+1. How to install PyTorch within Slicer. The main question is whether to install a version with GPU support and, if it does, which version of the CUDA toolkit to install.
+1. How to handle the necessary conversion of Slicer nodes (e.g., `vtkMRMLScalarNode`) to PyTorch objects (e.g., `torch.Tensor`) and vice versa. Look into adding tools to `slicer.util`.
```

Comment:
vtkMRMLScalarVolumeNode is what I mean here I think.

Reply:
Yes, thanks!